addon/authenticators/oauth2-password-grant.js

```diff
@@ -164,11 +161,19 @@ export default BaseAuthenticator.extend({
       }
       this.makeRequest(serverTokenEndpoint, data).then((response) => {
         run(() => {
+          this._validate(reject, response, 'access_token');
+          const refreshAccessTokens = this.get('refreshAccessTokens');
+          if (refreshAccessTokens) {
+            this._validate(reject, response, 'expires_in');
+            this._validate(reject, response, 'refresh_token');
```

Comment:
``` 
{ error: 'invalid_grant' }
```
is returned elsewhere if 'expires_in' and 'access_token' are already in the response. Maybe we can remove this?